frontend/packages/console-app/src/components/quick-starts/QuickStartPanelContent.tsx

```diff
@@ -32,6 +32,11 @@ const QuickStartPanelContent: React.FC<QuickStartPanelContentProps> = ({
   const shadows = useScrollShadows(contentRef);
 
   const quickStart = quickStarts.find((qs) => qs.metadata.name === activeQuickStartID);
+  const nextQuickStarts: QuickStart[] = [];
+  quickStart?.spec.nextQuickStart?.forEach((name) => {
+    const next = quickStarts.find((qs) => qs.metadata.name === name);
+    next && nextQuickStarts.push(next);
+  });
```

Comment:
can we have somethings as below?

```suggestion
  const nextQuickStarts: QuickStart[] = quickStarts.filter((qs:QuickStart) => 
     quickStart?.spec.nextQuickStart?.includes(qs.metadata.name));
```

Reply:
+1

Reply:
updated